hypothesis-python/docs/index.rst

```diff
@@ -104,17 +103,16 @@ Technical API reference.
   reference/index
   stateful
   Extras <extras>
-  compatibility
   changelog
 
 .. toctree::
   :maxdepth: 1
   :hidden:
-  :caption: Community
+  :caption: About Hypothesis
 
   development
+  compatibility
   usage
   extensions
```

Comment:
Let's re-title this page "Third-party Extensions".  Or maybe "Extensions & Plugins", and merge in the current Extras page.